tui/user_manager.go

```diff
@@ -182,13 +199,29 @@ func (page *UserManagerPage) redrawRows() {
 		curr.Destroy()
 	}
 	page.rowFrames = []*clui.Frame{}
+	adminSet := false
 
 	if len(page.users) > 0 {
 		for _, user := range page.users {
+			if user.Admin {
+				adminSet = true
+			}
 			if err := page.addUserRow(user); err != nil {
 				page.Panic(err)
 			}
 		}
+
+		// Display a warning when no admin user is set
+		if !adminSet {
+			rowFrame := clui.CreateFrame(page.scrollingFrame, 1, 1, clui.BorderNone, clui.Fixed)
```

Comment:
The warning message should be below the scrolling frame, not a part of it.
If you attempt to add 6 accounts, all non-admin, the warning will be hidden and not scrollable.

Reply:
Good catch, I'll fix this

Reply:
@mdhorn The scrollable frame is causing me issues. I created a new frame below the scrollable frame and placed the warning label inside. The scrollable frame pushes the newly created frame off of the screen. Even when I force the scrollable frame to use a small amount of space, the hidden (scrollable) elements still push the warning label lower... Any good ideas?

Reply:
@jwakre Make the scrollable frame one line smaller when it is created in `newUserManagerPage`.
Created the new frame for the warning label also in the `newUserManagerPage` function, and it should be added to the `page.content`, not the scrollable frame.

`page.warningFrame = clui.CreateFrame(page.content, 1, 1, clui.BorderNone, clui.Fixed)`

Reply:
I tried that approach. Even though there is plenty of screen real estate below the scrollable frame, as I add elements to the scrollable frame, the warning label in the newly created, page.content child frame is still pushed lower. It feels like a clui bug to me.

Reply:
@jwakre Yep, looks like a Clui bug.
I have an idea, give me a few minutes to try sometime.